cycledash/api/comments.py

```diff
@@ -163,6 +165,7 @@ def _row_key(comment, table):
             comment['last_modified'] = to_epoch(comment['last_modified'])
             comment['created'] = to_epoch(comment['created'])
             comment = camelcase_dict(comment)
+            comment = userify_comment(comment)
```

Comment:
Update this function's docstring to note that you're adding a "user" field to comments.